virtio-devices/src/balloon.rs

```diff
@@ -198,7 +302,10 @@ impl BalloonEpollHandler {
 
                 match queue_index {
                     0 => {
-                        Self::release_memory_range(desc_chain.memory(), range_base, range_len)?;
+                        if self.pbp.is_none() {
+                            self.pbp = Some(PartiallyBalloonedPage::new());
+                        }
```

Comment:
This can be done from ` Self::release_memory_range_4k()`.

Reply:
Yes